css/index.css

```diff
@@ -126,6 +157,30 @@ p {
     padding-right: 0;
 }
 
+/**==================================
+        Background colors
+====================================*/
+
+.bg-teal { background: teal }
+
+.bg-gold { background: gold }
+
+.bg-cadetblue { background: cadetblue }
+
+.bg-coral { background: coral }
+
+.bg-crimson { background: crimson }
+
+.bg-forestgreen { background: forestgreen }
+
+.bg-darkorchid { background: darkorchid }
+
+.bg-hotpink { background: hotpink }
+
+.bg-indigo { background: indigo }
+
+.bg-dodgerblue { background: dodgerblue }
+
```

Comment:
Very good with using the second class for the background colors. You could have used both classes to make sure it went to the box eg ```.box .bg-dodgerblue { background: dodgerblue }``` This would help if you had multiple items that would require the same background